mopidy_musicbox_webclient/static/js/controls.js

```diff
@@ -383,44 +383,27 @@ function doSingle() {
  * Use a timer to prevent looping of commands
  *********************/
 
-function doSeekPos(value) {
-    var val = $("#trackslider").val();
-    newposition = Math.round(val);
-    if (!initgui) {
-        pausePosTimer();
-        //set timer to not trigger it too much
-        clearTimeout(seekTimer);
-        $("#songelapsed").html(timeFromSeconds(val / 1000));
-        seekTimer = setTimeout(triggerPos, 500);
-    }
-}
-
-function triggerPos() {
-    if (mopidy) {
-        posChanging = true;
-        //        mopidy.playback.pause();
-        //    console.log(newposition);
-        mopidy.playback.seek(newposition);
-        //        mopidy.playback.resume();
-        resumePosTimer();
-        posChanging = false;
-    }
-}
-
-function setPosition(pos) {
-    if (posChanging) {
-        return;
-    }
+function timerCallback(position, duration) {
+    $("#songelapsed").html(timeFromSeconds(position / 1000));
```

Comment:
I would switch html with text so you don't waste time trying to convert plain text to HTML. Since this will be called from the browser animation loop we want to have everything as lean as possible.

Though as far as I can see text() would still set `textContent` so you will get UI trashing. Workaround would be to use the textNode trick as explained in the link bellow. As this text will only be changing about 1/10th of the time this gets called if you are running with 100ms between callbacks, and using the node trick the browser doesn't do anything expect when the text changes. While with `textContent` it always does work. (Yes this might be me over optimizing, so feel free to postpone this and measure once the change is in) 

http://stackoverflow.com/questions/17199958/why-does-setting-textcontent-cause-layout-thrashing